### DISCUSSION TITLE
WEB UI port

### DISCUSSION THREAD

**Discussion by arsikkx** [wg-easy/wg-easy#573] [Q&A] [answered]
how to change WEB UI port please help.
i want to run two instances of wg-easy, and i need to help to maintain it, seems like i found everything except web ui port changing

**Comment by eandersons** [comment, marked as answer]
Use port binding:

- with `docker run`: [`--publish` (`-p`)](https://docs.docker.com/engine/reference/commandline/run/#options);
- with Docker Compose: [`ports`](https://docs.docker.com/compose/compose-file/compose-file-v3/#ports).

With port binding container port(s) can be bound to any of available port(s) on host.

**Reply by Hutch79** [reply]
Do you happen to also know if it's possible to have two ports for one instance?
So one port for the UI (eg. 80) and one port for WireGuard?

Edit: I just found it... Sorry ^^'
